core/domain/exp_domain_test.py

```diff
@@ -507,14 +507,16 @@ def test_checkpoint_count_with_count_outside_range_is_invalid(self):
         self.exploration.validate()
 
     def test_bypassable_state_with_card_is_checkpoint_true_is_invalid(self):
+        # Note: In the graphs below, states  with * symbol are checkpoints.
```

Comment:
Drop space before "with"; add "the" after "with".

Reply:
Done